pkg/agent/run.go

```diff
@@ -170,7 +175,17 @@ func gatherAndOutputData(ctx context.Context, config Config, preflightClient *cl
 		}
 		log.Println("Data saved locally to", OutputPath)
 	} else {
-		postData(config, preflightClient, readings)
+		//TODO Change backoff parameters to those desired
```

Comment:
What do we want to do here? I think it's better to set something than to have a TODO, unless the defaults are fine, then I would just remove the TODO.